docs/examples/brep_overlap.py

```diff
@@ -26,14 +26,14 @@
 
 for face in FA[:1]:
     brep = BRep()
-    brep.occ_shape = face.occ_face
+    brep.native_brep = face.occ_face
```

Comment:
same here.
also, the viewmesh thing is no longer necessary, since breps are properly supported by the viewer